### ISSUE TITLE
Bookmark interesting turns for review later

### ISSUE THREAD

**Issue by misholson** [Zamiell/hanabi-live#1797]
There are frequently situations where I don't understand why a teammate made a certain play, and after the game we go through the game together and review the interesting spots.

It would be cool if there was a button that said "Bookmark", and if you click it the game saves that turn, and at the end when you're reviewing the turns there's an easy way to skip to the bookmarked turns.

It would be nice, but not necessary, to add a note to the bookmark to remind yourself what you thought was interesting.

Note: I'm not tied to the term "bookmark" but this is what my group used it say when we played in person when someone was confused and we wanted them to stop table-talking by being visibly confused. "Bookmark it!"

**Comment by padiwik** [comment]
Typically we use notes on the cards for this, which lets you add a turn number and whatever commentary you want (right-click a card...) I don't think jumping to a particular turn would be very useful for reviews, since you want to look at the context of previous turns while reviewing.

**Comment by misholson** [comment]
That makes sense. I'll start adding turn numbers to my notes and that will probably be enough of a reminder for me of whatever I wanted to discuss after the game.

I disagree with your point about context. Obviously you're right that the context of previous turns is important, but I think I'm going to start from the spot that I found interesting/confusing and then work backwards to see how we got there.

In any case I think you're right that notes are fine for this.
